Code/GraphMol/RGroupDecomposition/Wrap/rdRGroupComposition.cpp

```diff
@@ -350,7 +350,7 @@ struct rgroupdecomp_wrapper {
             &RDKit::RGroupDecompositionParameters::includeTargetMolInResults);
 
     python::class_<RDKit::RGroupDecompositionHelper, boost::noncopyable>(
-        "RGroupDecomposition", docString.c_str(),
+        "RGroupDecomposition", "",
```

Comment:
Previously the docstring defined for the `RGroupDecompositionParameters` was used here. I am not familiar with this case, so I simply put in an empty string.